datapipelines/src/main/kotlin/io/customer/datapipelines/plugins/CustomerIODestination.kt

```diff
@@ -79,13 +81,11 @@ class CustomerIODestination : DestinationPlugin(), VersionedPlugin, Subscriber {
         super.setup(analytics)
 
         // convert flushAt and flushIntervals into FlushPolicies
-        flushPolicies = if (analytics.configuration.flushPolicies.isEmpty()) {
+        flushPolicies = analytics.configuration.flushPolicies.ifEmpty {
             listOf(
                 CountBasedFlushPolicy(analytics.configuration.flushAt),
                 FrequencyFlushPolicy(analytics.configuration.flushInterval * 1000L)
             )
-        } else {
-            analytics.configuration.flushPolicies
```

Comment:
how are we handling this case after the change?

Reply:
this was an outdated change from stacked PR.